packages/vector_graphics/lib/src/listener.dart

```diff
@@ -3,6 +3,21 @@ import 'dart:typed_data';
 
 import 'package:vector_graphics_codec/vector_graphics_codec.dart';
 
+/// The deocded result of a vector graphics asset.
+class PictureInfo {
+  /// Construct a new [PictureInfo].
+  const PictureInfo(this.picture, this.viewBox);
+
+  /// The picture to be drawn with [ui.canvas.drawPicture]
+  final ui.Picture picture;
+
+  /// The target size and position of the picture.
+  ///
+  /// This information should be used to scale and position
+  /// the picture based on the available space and alignment.
+  final ui.Rect viewBox;
```

Comment:
In the end we should really just need a size here. the rect would be to specify an additional translate that we pre-apply

Reply:
Ahh yeah I was sort of wondering about that. Makes the encoding slightly smaller too.

Can we always pre-translate it in the compiler? (Assuming that is what you mean)

Reply:
Yes. The feature is also rarely used, 99+% of SVGs use the origin as 0,0

Reply:
Done